rct_optimizations/src/rct_optimizations/pnp.cpp

```diff
@@ -53,16 +53,15 @@ struct SolvePnPCostFunc3D
   {}
 
   template<typename T>
-  bool operator()(const T *const target_q, const T *const target_t, T *const residual) const
+  bool operator()(const T *const target_aa, const T *const target_t, T *const residual) const
   {
-
     using Isometry3 = Eigen::Transform<T, 3, Eigen::Isometry>;
     using Vector3 = Eigen::Matrix<T, 3, 1>;
 
-    Eigen::Map<const Eigen::Quaternion<T>> q(target_q);
+    Eigen::Map<const Vector3> aa(target_aa);
```

Comment:
Nitpicky style preference thing, but I'd prefer if `aa` and `t` were named more verbosely so they can be understood at a glance without needing to follow them through the code.